spec/features/case_management/tax_return_status_change_spec.rb

```diff
@@ -5,32 +5,36 @@
     let(:vita_partner) { create :vita_partner }
     let(:user) { create :user, name: "Example Preparer", vita_partner: vita_partner }
     let(:client) { create :client, vita_partner: vita_partner }
-    let!(:intake) { create :intake, client: client }
+    let!(:intake) { create :intake, client: client, locale: "en" }
     let!(:tax_return) { create :tax_return, year: 2019, client: client, status: "intake_in_progress" }
 
     before do
       login_as user
     end
 
-    scenario "logged in user can change a status on a tax return" do
+    xscenario "logged in user can change a status on a tax return" do
```

Comment:
Is this skipped because the PR is not ready yet? Just curious to understand.

Reply:
I stopped skipping it.